package.json

```diff
@@ -20,7 +20,8 @@
   "scripts": {
     "watch": "tsc --watch",
     "build": "tsc",
-    "lint": "tslint --project tsconfig.json --type-check --format stylish"
+    "lint": "tslint --project tsconfig.json --type-check --format stylish",
+    "test": "echo test/* test/*/* | xargs -n 1 tslint --test"
```

Comment:
Is this expected to be runnable on Win10?
Note that `xargs` doesn't exist on Windows and `echo` behaves differently there as well. It doesn't look like powershell or cmd expand `echo test/*`. Instead the output on Windows is `test/*`.

With bash on Ubuntu, `test/*` get's expanded into all the corresponding entries. I'm not sure whether `tslint` accepts a wildcard path or a list of files or both, but this is something to be aware of.